src/ui/components/Views/DevView.tsx

```diff
@@ -47,8 +44,14 @@ function DevView({
         <SplitBox
           style={{ width: "100%", overflow: "hidden" }}
           splitterSize={8}
-          initialSize={prefs.toolboxHeight as string}
-          minSize={showEditor || !sidePanelCollapsed ? "240px" : "0%"}
+          initialSize={prefs.toolboxSize as string}
+          minSize={
+            !sidePanelCollapsed && showEditor
```

Comment:
Nested ternary is 😬. Maybe a helper similar to `maxSize()` that looks something like this?

```js
function minSize(...) {
  if (!sidePanelCollapsed) {
    if (showEditor) {
      return "260px"
    } else {
      return "200px"
    }
  } else {
    return "0%"
  };
}
```